buildenv/jenkins/variables/defaults.yml

```diff
@@ -163,11 +163,12 @@ boot_jdk_default:
       17: '17'
       19: '18'
       20: '18'
-      next: '18'
+      next: '20'
     dir_strip:
       all: '1'
     url:
-      all: "https://api.adoptopenjdk.net/v3/binary/latest/${bootJDKVersion}/ga/${os}/${arch}/jdk/openj9/normal/adoptopenjdk?project=jdk"
+      all: 'https://api.adoptopenjdk.net/v3/binary/latest/${bootJDKVersion}/ga/${os}/${arch}/jdk/openj9/normal/adoptopenjdk?project=jdk'
+      next: 'https://api.adoptium.net/v3/binary/latest/${bootJDKVersion}/ga/${os}/${arch}/jdk/hotspot/normal/eclipse?project=jdk'
```

Comment:
I think `next` needs to be `20`

Reply:
Ah, I think I see what you mean; '20': {url} - that makes sense.

Reply:
Updated in 93021cff0d186f56ab97ff9d978dcd57eec7922d.